functions/stringify/1.0/function.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Stringify a value",
+  "label": "Stringify",
+  "category": "Misc",
+  "icon": {
+    "name": "TextIcon",
+    "color": "Purple"
+  },
+  "options": [{
```

Comment:
Make both options required value and result

Reply:
added